db/classManagement.go

```diff
@@ -0,0 +1,305 @@
+package db
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"../tools/requests"
+)
+
+// HandleCreateClass is the handler for creating a new class.
+// It takes the UID of the creator, the name of the class, and a thunmbnail id. 
+func (d *DB) HandleCreateClass(w http.ResponseWriter, r *http.Request) {
+	
+	var (
+		err error
+	)
+
+	//create an anonymous structure to handle requests
+	req := struct {
+		UID 		string  	`json:"uid"`
+		Name 		string		`json:"name"`
+		Thumbnail 	int64 		`json:"thumbnail"`
+	}{}
+	//read JSON from request body
+	if err = requests.BodyTo(r, &req); err != nil {
+		http.Error(w, "Error occurred in reading body.", http.StatusInternalServerError)
+		return
+	}
+
+	if req.UID == "" {
+		http.Error(w, "Error occurred in reading UID.", http.StatusInternalServerError)
+		return
+	}
+	
+	if req.Name == "" {
+		http.Error(w, "Error occurred in reading Name.", http.StatusInternalServerError)
+		return
+	}
+
+	if req.Thumbnail < 0 || req.Thumbnail >= 50 {
```

Comment:
We have a thumbnail count constant defined already in `constants.go` if I remember correctly. Make sure to use that one, unless there is an alternative thumbnail limit in the JavaScript backend.